.gitignore

```diff
@@ -1 +1 @@
-*.png
```

Comment:
I think we should keep `png` files in the gitignore to prevent the repository from becoming too big. If you feel a few pngs are really necessary you can force add them to a commit.